core/lib/protobuf_config/src/proto/config/general.proto

```diff
@@ -2,29 +2,30 @@ syntax = "proto3";
 
 package zksync.config.general;
 
-import "zksync/config/prover.proto";
 import "zksync/config/api.proto";
+import "zksync/config/base_token_adjuster.proto";
```

Comment:
avoid unnecessary change to make cherry picking easier 🙏

Reply:
done

Reply:
most of the import are still moved

Reply:
Could you give me the link? I don't see the import moved.
https://github.com/cronos-labs/cronos-zkevm/pull/79/files#diff-6fd8304f2e245f1c6b25b6840fdf490524151f74810afa04cdfe4af4d60810a0

Reply:
ah nvm maybe I was looking outdated version, sorry